tests/facts/windows_computer_info/info.json

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
Could we trim this test down a bit too?

Reply:
I think we want to leave this one "as-is".